prow/proxy-postsubmit.sh

```diff
@@ -32,6 +32,16 @@ if [[ -n "${GOOGLE_APPLICATION_CREDENTIALS:-}" ]]; then
   gcloud auth configure-docker
 fi
 
+# Override envoy.
+if [[ "${ENVOY_REPOSITORY:-}" && "${ENVOY_SHA:-}" && "${ENVOY_PREFIX:-}" ]]; then
```

Comment:
This should be done in `prow/proxy-common.inc` so that it would apply to all the jobs, not only postsubmit.

Reply:
Moved logic to `prow/proxy-common.inc`.